daft/daft.pyi

```diff
@@ -1251,6 +1239,26 @@ def image_encode(expr: PyExpr, image_format: ImageFormat) -> PyExpr: ...
 def image_resize(expr: PyExpr, w: int, h: int) -> PyExpr: ...
 def image_to_mode(expr: PyExpr, mode: ImageMode) -> PyExpr: ...
 
+# expr.float namespace
+def is_nan(expr: PyExpr) -> PyExpr: ...
+def is_inf(expr: PyExpr) -> PyExpr: ...
+def not_nan(expr: PyExpr) -> PyExpr: ...
+def fill_nan(expr: PyExpr, fill_value: PyExpr) -> PyExpr: ...
+
+# expr.json namespace
+def json_query(expr: PyExpr, query: str) -> PyExpr: ...
+
+# expr.list namespace
+def list_chunk(expr: PyExpr, size: int) -> PyExpr: ...
+def list_count(expr: PyExpr, mode: CountMode) -> PyExpr: ...
+def list_join(expr: PyExpr, delimiter: PyExpr) -> PyExpr: ...
+def list_get(expr: PyExpr, idx: PyExpr, default: PyExpr) -> PyExpr: ...
+def list_sum(expr: PyExpr) -> PyExpr: ...
+def list_mean(expr: PyExpr) -> PyExpr: ...
+def list_min(expr: PyExpr) -> PyExpr: ...
+def list_max(expr: PyExpr) -> PyExpr: ...
+def list_slice(expr: PyExpr, start: PyExpr, end: PyExpr | None = None) -> PyExpr: ...
+
```

Comment:
Clean! Like this change a lot, especially with the transition to `daft-functions`.